Install failed with pnpm and @react-pdf/renderer 2.3.0

andrei:atlas imax$ pnpm i
 WARN  deprecated babel-eslint@10.1.0: babel-eslint is now @babel/eslint-parser. This package will no longer receive updates.
 WARN  deprecated querystring@0.2.1: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
 WARN  deprecated source-map-resolve@0.6.0: See https://github.com/lydell/source-map-resolve#deprecated
Packages: +50 -48
++++++++++++++++++++++++++++++++++++++++++++++++++------------------------------------------------
Progress: resolved 1799, reused 1280, downloaded 2, added 2, done
node_modules/.pnpm/vite-plugin-shim-react-pdf@1.0.6_2c55icevksybp3b2dgwz2fpj5y/node_modules/vite-plugin-shim-react-pdf: Running postinstall script, failed in 124ms
.../vite-plugin-shim-react-pdf postinstall$ node scripts/shim-react-pdf.js
│  - Skipping 'blob-stream', already patched...
│ Patching '@react-pdf/pdfkit/lib/pdfkit.browser.es.js'...
│ Patching '@react-pdf/fontkit/lib/fontkit.browser.es.js'...
│  - Skipping '@react-pdf/png-js/lib/png-js.browser.es.js', already patched...
│ node:internal/modules/cjs/loader:949
│   throw err;
│   ^
│ Error: Cannot find module '@react-pdf/unicode-properties/lib/unicode-properties.es.js'
│ Require stack:
│ - /Users/imax/WebstormProjects/atlas/node_modules/.pnpm/vite-plugin-shim-react-pdf@1.0.6_2c55icevksybp3b2dgwz2fpj5y/node_modules/vite-plugin-shim-react-pdf/scripts/shim-react-pdf.js
│     at Module._resolveFilename (node:internal/modules/cjs/loader:946:15)
│     at Function.resolve (node:internal/modules/cjs/helpers:108:19)
│     at prependFiles (/Users/imax/WebstormProjects/atlas/node_modules/.pnpm/vite-plugin-shim-react-pdf@1.0.6_2c55icevksybp3b2dgwz2fpj5y/node_modules/vite-plugin-shim-react-pdf/scripts/shim-react-pdf.js:25:26)
│     at Object.<anonymous> (/Users/imax/WebstormProjects/atlas/node_modules/.pnpm/vite-plugin-shim-react-pdf@1.0.6_2c55icevksybp3b2dgwz2fpj5y/node_modules/vite-plugin-shim-react-pdf/scripts/shim-react-pdf.js:64:1)
│     at Module._compile (node:internal/modules/cjs/loader:1112:14)
│     at Module._extensions..js (node:internal/modules/cjs/loader:1166:10)
│     at Module.load (node:internal/modules/cjs/loader:988:32)
│     at Module._load (node:internal/modules/cjs/loader:834:12)
│     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
│     at node:internal/main/run_main_module:17:47 {
│   code: 'MODULE_NOT_FOUND',
│   requireStack: [
│     '/Users/imax/WebstormProjects/atlas/node_modules/.pnpm/vite-plugin-shim-react-pdf@1.0.6_2c55icevksybp3b2dgwz2fpj5y/node_modules/vite-plugin-shim-react-pdf/scripts/shim-react-pdf.js'
│   ]
│ }

--- a/scripts/shim-react-pdf.js
+++ b/scripts/shim-react-pdf.js
@@ -66,7 +66,6 @@ prependFiles(
     "@react-pdf/pdfkit/lib/pdfkit.browser.es.js",
     "@react-pdf/fontkit/lib/fontkit.browser.es.js",
     "@react-pdf/png-js/lib/png-js.browser.es.js",
-    "@react-pdf/unicode-properties/lib/unicode-properties.es.js",
   ],
   "import { Buffer } from 'buffer';"
 );

